store/migrate.go

```diff
@@ -81,3 +82,17 @@ func migrateToV2(tx *sql.Tx) error {
 	}
 	return nil
 }
+
+func migrateToV3(tx *sql.Tx) error {
+	for _, t := range []string{
+		"ALTER TABLE aciinfo ADD name string",
```

Comment:
For context: ql does not support renaming columns, and neither does it support `ALTER TABLE RENAME TO`, so I could not copy the table and create a new one. Happy to hear if anyone else has alternative solutions for not re-ordering the columns...